docroot/config/sync/core.entity_view_display.paragraph.image.default.yml

```diff
@@ -17,7 +17,7 @@ content:
     weight: 0
     label: hidden
     settings:
-      image_style: large
+      image_style: content_image_style_670x480_
```

Comment:
The is a space in the machine name of the image style which translates into a _ in it.